README.md

```diff
@@ -13,46 +13,47 @@ Znaczy [pull requestów](https://github.com/nurkiewicz/polski-w-it/pulls), nie t
 
 | angielski            | polskawy            | polski                                |
 | :------------------- | :------------------ | :------------------------------------ |
-| approve              | zapruwować          | zaakceptować                          |
+| approve              | zaapruwować         | zaakceptować                          |
 | authentication       | autentykacja        | uwierzytelnienie (!)                  |
 | authorization        |                     | autoryzacja                           |
 | build                | bild                | budowanie                             |
-| button               | baton               | przycisk                              |
-| cancel               | kanselować          | anulować                              |
+| button               | baton               | przycisk, guzik                       |
+| cancel               | kanselować, anulacja| anulować                              |
 | case                 | kejs                | przypadek                             |
 | confirm              | konfirmować         | potwierdzić                           |
-| content              |                     | zawartość, treść                      |
+| content              | kontent             | zawartość, treść                      |
 | custom               | kastomowy           | specjalny, nietypowy, niestandardowy  |
 | default              | difoltowo           | domyślnie                             |
 | deploy               | zdiplojować         | wdrożyć                               |
 | developer            |                     | programista                           |
-| edge case            |                     | przypadek brzegowy                    |
-| event                |                     | zdarzenie                             |
-| feature              | ficzer              | funkcja                               |
-| feedback             |                     | informacja zwrotna                    |
+| edge case            | edż kejs            | przypadek brzegowy                    |
+| event                | iwent               | zdarzenie, wydarzenie                 |
+| feature              | funkcjonalność, ficzer| funkcja                             |
```

Comment:
:-1:, chyba nie mogę się zgodzić. Jeśli już to _functionality_ jest niepoprawnie tłumaczone jako _funkcjonalność_.

Reply:
Najczęściej słyszymy funkcjonalność jako feature.
Ale cofam zmianę w tym wierszu, żeby inne w końcu weszły.